website/www/site/content/en/get-started/beam-overview.md

```diff
@@ -66,7 +66,7 @@ Get started using Beam for your data processing tasks.
 > If you already know [Apache Spark](https://spark.apache.org/),
 > check our [Getting started from Apache Spark](/get-started/from-spark) page.
 
-1. Take the [Tour of Beam](/get-started/tour-of-beam) as an online interactive learning experience.
+1. Get an [Interactive Overview of Beam](/get-started/an-interactive-overview-of-beam) as an online interactive learning experience.
```

Comment:
Can we actually leave this one as it was previously? I think tour of beam is a better thing for this to point to anyways